x-pack/plugin/esql/qa/testFixtures/src/main/resources/stats.csv-spec

```diff
@@ -3463,3 +3463,77 @@ employees:long | job_positions:keyword
 15             | Tech Lead    
 ;
 
+fixClassCastBug
+required_capability: fix_stats_classcast_exception
+
+from airports 
+| rename scalerank AS x 
+| stats  a = count(x), b = count(x) + count(x), c = count_distinct(x)
+;
+
+a:long | b:long | c:long
+891    | 1782   | 8
+;
+
+fixClassCastBug2
```

Comment:
Nit: It would be nice to avoid numbers in test cases.
Additional description could also hint what aspects were broken before:
* combining results of two aggregate functions
* nesting functions
* multiplying result by constant
* etc